src/main/cli/settings.c

```diff
@@ -533,9 +537,10 @@ const lookupTableEntry_t lookupTables[] = {
     LOOKUP_TABLE_ENTRY(lookupTableUnit),
     LOOKUP_TABLE_ENTRY(lookupTableAlignment),
 #ifdef USE_GPS
-    LOOKUP_TABLE_ENTRY(lookupTableGPSProvider),
-    LOOKUP_TABLE_ENTRY(lookupTableGPSSBASMode),
-    LOOKUP_TABLE_ENTRY(lookupTableGPSUBLOXModels),
+    LOOKUP_TABLE_ENTRY(lookupTableGpsProvider),
+    LOOKUP_TABLE_ENTRY(lookupTableGpsSbasMode),
+    LOOKUP_TABLE_ENTRY(lookupTableGpsUbloxModels),
```

Comment:
Isn't it used for `gps_ublox_flight_model` and `ps_ublox_aquire_model` in settings?

Reply:
> Isn't it used for `gps_ublox_flight_model` and `ps_ublox_aquire_model` in settings?

Yes, all of those are actual user-configurable settings that are used.

afaik, the only cli GPS setting we aren't currently using is `gps_auto_baud`, which is ignored while we figure out what it means and whether it can work as intended.